js/woo.js

```diff
@@ -24,7 +24,7 @@ module.exports = class woo extends Exchange {
                 'spot': true,
                 'margin': true,
                 'swap': false,
-                'future': false,
+                'future': true,
```

Comment:
```
‘future’: false,
‘swap’: true,
```
All of the futures trading pairs are perpetual futures (what we call swap) future should only be set to true if there are traditional delivery futures with an expiration date available. Set future to false and set swap to true.